styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,47 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container hidden-xs hidden-sm">
   <div class="row">
     <div class="col-sm-3">
-      <div class="col-sm-3">
-        <h5>We've got your back</h5>
-        <h4 class="text34">24x7x365</h4>
+      <h5 class="rug-subhead-medium no-margin">We've got your back</h5>
+      <h4 class="rug-subhead-large no-margin">24<small class="white">x</small>7<small class="white">x</small>365</h4>
```

Comment:
Not a fan of using `.white` here to override colors set for all `small` elements within `h4` but that is what a flexible modifier class provides. In the future we need to refactor our CSS to not style elements but instead style classes and let styles cascade properly. In this case, if the `small` elements didn't have styles applied to them then they could inherit their color from `.rug-subhead-large`. Just because something is `small` doesn't mean it has to have a specific color, so it is better to leave styling to the outer element or apply a class for _explicit_ styling when needed.